cypress/e2e/dfeAdmin/addServices/addServices-dfeAdmin-ServiceAdded.spec.js

```diff
@@ -0,0 +1,135 @@
+describe("LA Manager - Add Services - Service Added Page", () => {
```

Comment:
Should the describe be Dfe Admin instead of LA manager?

Reply:
Yes it should, resolved in [73cfda5](https://github.com/DFE-Digital/fh-admin-service-testing/pull/140/commits/73cfda5a293cba31a7954d1b2af875182567537c)

(also checked vcs manager test name and sorted that)